code/modules/mob/hear_say.dm

```diff
@@ -72,7 +76,7 @@
 				to_chat(src, "<span class='name'>[speaker_name]</span>[alt_name] talks but you cannot hear \him.")
 	else
 		if(language)
-			on_hear_say("[track]<span class='game say'><span class='name'>[speaker_name]</span>[alt_name] [language.format_message(message, verb)]</span>")
+			on_hear_say("[track][accent_icon]<span class='game say'><span class='name'>[speaker_name]</span>[alt_name] [language.format_message(message, verb)]</span>")
```

Comment:
I'd put a space after the icon, personally? It would look better and make letters like J/D blend in less.